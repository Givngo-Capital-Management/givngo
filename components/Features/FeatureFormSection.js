import React from 'react';
import styles from './FeatureFormSection.module.scss';
import FeatureForm from './FeatureForm';

export default function FeatureFormSection() {
  return (
    <div id="Form-section" className={styles['form-container']}>
      <div className={styles['form-text-container']}>
        <div className={styles['form-text-wrapper']}>
          <div className={styles['text-container']}>
            <h1 className={styles.header}>Marketing and Publishing</h1>
            <p className={styles.paragraph}>
              We will publish your videos and ads to the highest converting
              audiences possible. We format videos to fit all major platforms
              like Facebook, Google, Youtube, Vimeo, Instagram and custom sites.
              Givngo success managers create highly targeted campaigns that
              provide live feedback maximizing your marketing ROI.
            </p>
            <ul className={styles.list}>
              <li>
                <div>We create videos, ads, promotions</div>
              </li>
              <li>
                <div>Run targeted marketing campaigns</div>
              </li>
              <li>
                <div>Grow your charity donations</div>
              </li>
            </ul>
            <div className={styles.arrow}>
              <img
                src="https://givngo.io/wp-content/themes/givngo/images/right-arrow-1.svg"
                width="60"
                alt="right-arrow"
                className={styles.image}
              />
            </div>
          </div>
          <FeatureForm />
        </div>
      </div>
    </div>
  );
}
