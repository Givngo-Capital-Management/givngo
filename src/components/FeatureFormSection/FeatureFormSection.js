import React from 'react'
import './FeatureFormSection.scss'
import FeatureForm from '../FeatureForm/FeatureForm'

export default function FeatureFormSection() {
  return (
    <div id="Form-section" className="form-container">
      <div className="form-text-container">
        <div className="form-text-wrapper">
          <div className="text-container">
            <h1 className="header">Marketing and Publishing</h1>
            <p className="paragraph">We will publish your videos and ads to the highest converting audiences possible. We format videos to fit all major platforms like Facebook, Google, Youtube, Vimeo, Instagram and custom sites. Givngo success managers create highly targeted campaigns that provide live feedback maximizing your marketing ROI.</p>
            <ul className="list">
              <li>
                <div>We create videos, ads, promotions</div>
              </li>
              <li>
                <div>Run targeted marketing campaigns</div>
              </li>
              <li>
                <div>Grow your charity donations</div>
              </li>
            </ul>
            <div className="arrow">
              <img src="https://givngo.io/wp-content/themes/givngo/images/right-arrow-1.svg" width="60" alt className="image" />
            </div>
          </div>
          <FeatureForm />
        </div>
      </div>
    </div>
  )
}