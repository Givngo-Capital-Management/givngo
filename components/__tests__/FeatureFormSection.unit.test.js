import React from 'react';
import { shallow } from 'enzyme';
import FeatureFormSection from '../Features/FeatureFormSection';

describe('FeatureFormSection', () => {
  test('renders', () => {
    const wrapper = shallow(<FeatureFormSection />);
    expect(wrapper).toMatchSnapshot();
  });
});
