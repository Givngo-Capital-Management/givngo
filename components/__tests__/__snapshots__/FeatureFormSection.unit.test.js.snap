// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeatureFormSection renders 1`] = `
<div
  id="Form-section"
>
  <div>
    <div>
      <div>
        <h1>
          Marketing and Publishing
        </h1>
        <p>
          We will publish your videos and ads to the highest converting audiences possible. We format videos to fit all major platforms like Facebook, Google, Youtube, Vimeo, Instagram and custom sites. Givngo success managers create highly targeted campaigns that provide live feedback maximizing your marketing ROI.
        </p>
        <ul>
          <li>
            <div>
              We create videos, ads, promotions
            </div>
          </li>
          <li>
            <div>
              Run targeted marketing campaigns
            </div>
          </li>
          <li>
            <div>
              Grow your charity donations
            </div>
          </li>
        </ul>
        <div>
          <img
            alt="right-arrow"
            src="https://givngo.io/wp-content/themes/givngo/images/right-arrow-1.svg"
            width="60"
          />
        </div>
      </div>
      <FeatureForm />
    </div>
  </div>
</div>
`;
